content/guides/anomalies.md

```diff
@@ -125,9 +130,11 @@ Smaller window sizes will lead to faster alerts, however, with very small window
 
 Note that setting the window size to X minutes doesn't require an anomaly to last X minutes before an alert is triggered. You can tune the threshold to control how long an anomaly must last to trigger an alert. For example, with the window size set to 30 minutes, you can get alerted when an anomaly lasts for just five minutes by setting the threshold to 5/30 = 17%. That said, we have found that anomaly alerts are most reliable when the window size is between 15 minutes and an hour and the threshold is on the higher side (> 40%).
 
-### Why does `anomalies` not add a gray prediction band in the dashboard? Why am I getting "No Data" for an Anomaly Alert?
+### Why does `anomalies` not add a gray prediction band in the dashboard?
+### Why am I getting "No Data" for an Anomaly Alert?
+### How much history do the algorithms require?
 
-All the algorithms besides _Basic_ require historical data before they can start making predictions. If your metric has only started reporting data for a short while, then _Agile_ and _Robust_ won't try to make any predictions until it has a couple weeks of history. _Adaptive_ will start working after it has a couple days worth of history.
+All the algorithms besides _Basic_ require historical data before they can start making predictions. If your metric has only started reporting data for a short while, then _Agile_ and _Robust_ won't try to make any predictions until it has at least two weeks of history. _Adaptive_ will start working after it has at least 2 hours worth of history.
```

Comment:
As discussed, this is a bit confusing w.r.t. to the picture above that shows anomaly bounds starting after more than 2 hours worth of history.
Otherwise, looks good to me.